### PR TITLE
Add a sunset procedure for SIGs/WGs.

### PR DESCRIPTION
**Pull request by castrojo**
Initial fix for #2029

### CODE REVIEW

**Comment by castrojo** [comment]
/hold

We shouldn't have two `sig-governance.md` documents.

**Comment by castrojo** [comment]
Possible solution would be to rename this `sig-lifecycle.md` and specify that the document is more about day-to-day execution whereas the sig-governance.md document in the steering directory is more about governance.

**Comment by spiffxp** [comment]
I would largely agree except for the top section, there is overlap between that and the sig-governance template when it comes to spelling out roles and responsibilities.  I will see if there's someone on steering who wants to take on resolving the overlap, and then I think it could be a much more shortened list with links.

**Comment by castrojo** [comment]
Ok I've addressed Paris' review comments, standing by for the governance demarkation discussion.

**Comment by parispittman** [comment]
@cblecker - is there anything else we are missing from a github managememt perspective?

File: sig-governance.md [hunk 1]
```diff
@@ -67,3 +67,26 @@ Each SIG must have two discussion groups with the following settings.
...
+Working Groups in particular are more ephemeral than SIGs, so this process should be followed when the Working Group has accomplished it's mission.
+
+Retiring a SIG for is covered in the [SIG Governance](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)
+Retiring a Working Group is covered in [WG Governance](https://github.com/kubernetes/community/blob/master/committee-steering/governance/wg-governance.md)
+
+The process for closing a SIG/WG is as follows:
+
+- SIG Chairs agree to disband. This decision should follow the decision making process of the SIG's Charter.
+- 
+- Work with SIG Contributor Experience to:
```
[the remark this answers is not in the document]
Comment (cblecker) [reply sig-governance.md]: Please add the following tasks:
- "Clean up and remove all GitHub teams that refer to the sig"
- "Migrate/Remove/Mark as deprecated any sig labels in [labels.yaml](https://git.k8s.io/test-infra/label_sync/labels.yaml)"

Anything else I'm forgetting @spiffxp @fejta ?

[the remark this answers is not in the document]
Comment (spiffxp) [reply sig-governance.md]: lgtm

File: sig-governance.md [hunk 2]
```diff
@@ -55,15 +55,41 @@ Create a Google Group at [https://groups.google.com/forum/#!creategroup](https:/
...
+
+- SIG Chairs agree to disband. This decision should follow the decision making process of the SIG's Charter.
+- Send a email to kubernetes-dev to let people know the SIG has either closed or merged with another SIG. This will let SIG Contributor Experience know that they need to help you archive/deactivate project resources.
+  - Consider sending a [closing summary](https://docs.google.com/document/d/1qZcAvuWBznR_oEaPWtwm7U4JNT91m8r9YOUvInU-src/edit#heading=h.jsw0l2t0ra8) to the list. 
+- Work with SIG Contributor Experience to:
+  - Archive the mailing list/group
+  - Archive the leads mailing list/group
+  - Archive the slack channel
+  - Deactivate the group's Zoom license
+  - Move all appropriate github repositories to an appropriate archive or a repo outside of the Kubernetes org
```
Comment (spiffxp) [inline sig-governance.md]: For each subproject the SIG owns, either transfer ownership of the subproject to another SIG, or outside of the project, or retire the subproject.  If the subproject contains one more entire repos and is to be retired, file an issue on kubernetes/org to retire the repo

Comment (spiffxp) [reply sig-governance.md]: (As I write this out I wonder if maybe these assets should be sig subprojects, but probably just add as bullets for now)

- Retire or transfer any jobs owned by the sig (kubernetes/test-infra/config/jobs/kubernetes/sig-foo, or kuberentes/test-infra/config/jobs/orgname/reponame/jobs.yaml for repos owned by the sig)
- Retire or transfer any testgrid dashboards owned by the sig

**Comment by spiffxp** [comment]
re: the overlap in the top section, I won't block on that, and will address in a followup https://github.com/kubernetes/community/issues/2857

**Comment by spiffxp** [comment]
/cc @dims 
if you had to guinea pig this as written, is there anything missing from your perspective?

**Comment by parispittman** [comment]
we can rename slack channels @dims

File: sig-governance.md [hunk 3]
```diff
@@ -55,15 +55,46 @@ Create a Google Group at [https://groups.google.com/forum/#!creategroup](https:/
...
+
+The process for closing a SIG/WG is as follows:
+
+- SIG Chairs agree to disband. This decision should follow the decision making process of the SIG's Charter.
+- Send a email to kubernetes-dev to let people know the SIG has either closed or merged with another SIG. This will let SIG Contributor Experience know that they need to help you archive/deactivate project resources.
+  - Consider sending a [closing summary](https://docs.google.com/document/d/1qZcAvuWBznR_oEaPWtwm7U4JNT91m8r9YOUvInU-src/edit#heading=h.jsw0l2t0ra8) to the list. 
+- Work with SIG Contributor Experience to:
+  - Archive the mailing list/group
+  - Archive the leads mailing list/group
+  - Archive the slack channel
```
[the remark this answers is not in the document]
Comment (castrojo) [reply sig-governance.md]: Yes, in fact we usually do renames for slack channels as it's least disruptive for users.

**Comment by spiffxp** [comment]
Brian expressed some concerns that WG/SIG's in the past that have been sunset ended up losing docs since the googlegroup they were shared with was decomissioned.  Want to make sure however docs are archived, it's done so in such a way that we handle this case. (I haven't read through this PR with that in mind, so maybe this is addressed, just braindumping during steering meeting)

**Comment by parispittman** [comment]
Decommissioning a mailing list here means that we turn off the ability to post and allow for new members. The group itself does not go away and therefore it's docs wouldn't either. @spiffxp

**Comment by castrojo** [comment]
Any other actions or comments? I feel like this one is ready to go!

**Comment by dims** [comment]
LGTM 👍

**Comment by cblecker** [comment]
LGTM

**Comment by jdumars** [comment]
/approve
(a steering committee member can lift the hold to merge)

**Pull request merged**